### DISCUSSION TITLE
Maybe add search functionality and a back to top button to the Terminal theme.

### DISCUSSION THREAD

**Discussion by SimpleDIY** [GetPublii/Publii#1571] [Ideas]
I really like the Terminal theme. I just wish it had a back to top button (like other Publii themes) and the search functionality. Then I could also buy the static search plugin to support Publii ;)

**Comment by bttrx** [comment]
Today I crafted a back to top button for https://www.robertriebisch.de/ using code from the also free Tattoo theme. I could offer you a template override for Terminal version 2.1.0.0, if you are still interested.

@dziudek or @bobmitro Something for the next Terminal release?

**Reply by bobmitro** [reply]
What do you mean, what's wrong with this theme?

**Reply by bttrx** [reply]
Terminal 2.1.0.0 lacks a 'back to top' button, which I added now for my own site. But I would share the code with you, so anyone could benefit.

**Reply by bobmitro** [reply]
Unfortunately, I don't have time for requests like this. I also don't consider this feature particularly important. If you have the code, you can post it here so others can benefit.

And please don't mention me on GitHub in threads like this.

**Reply by rgblack316** [reply]
That's a pretty flippant attitude to have towards an end user of the project.

**Comment by bttrx** [comment]
[backtotop.zip](https://github.com/user-attachments/files/18223162/backtotop.zip) holds a [theme override](https://getpublii.com/dev/theme-overrides/) for Terminal theme version 2.1.0.0 to provide a back to top button. Archive content is meant to be placed to the `Publii/sites/your-siteName/input/` folder.

What's inside:
```
.
├── languages
│   └── terminal.lang.json
└── themes
    └── terminal-override
        ├── assets
        │   ├── css
        │   │   └── main.css
        │   ├── js
        │   │   ├── scripts.js
        │   │   └── scripts.min.js
        │   └── svg
        │       └── svg-map.svg
        ├── config.json
        └── partials
            └── footer.hbs
```
